cmd/identity/batch.go

```diff
@@ -111,7 +111,7 @@ func saveIdentityTar(path string, key crypto.PrivateKey, id storj.NodeID) error
 		return err
 	}
 
-	caCert, err := peertls.NewCert(key, nil, ct, nil)
+	caCert, err := peertls.NewSelfSignedCert(key, ct)
```

Comment:
We definitely should - all (hopefully) the bad variable names are from old commits and I wanted to avoid adding more to the  PR. I'm not sure if it's worth making a ticket for but it might be a good idea to just do a PR with variable name improvements in a bunch of places to clean this sort of thing up.